urwid/main_loop.py

```diff
@@ -1468,6 +1468,8 @@ def _exception_handler(self, loop, context):
                 # Store the exc_info so we can re-raise after the loop stops
                 import sys
                 self._exc_info = sys.exc_info()
+                if self._exc_info == (None,None,None):
+                    self._exc_info=exc
```

Comment:
Can you please add whitespaces around operators & commas in 1471-1472? Thanks!
Everything else looks great.

Reply:
Will do. I saw that was still in there from copying the original code, but not until after I submitted the PR.
(Didn't want to add any more commit noise than necessary)

Reply:
Also, would it be a good idea to add some reference info as a comment?